src/index.css

```diff
@@ -1,3 +1,9 @@
+body,
+html,
+#root {
+    height: 100%;
```

Comment:
Just curious, how this change corresponds to the same style applied in **src/components/AppWrapper/AppWrapper.tsx** ?
```
 root: {
    height: '100%',
    [theme.breakpoints.down('sm')]: {
      paddingTop: theme.spacing(7),
    },
```

Reply:
It's 2 different styles, first in global `.css` is applied only to `body` `html` and `#root` this last one is just the div where react loads the app.
And the `appWrapper root` refers to that specific component. 
I mean to height 100% all parents components also need to have 100% that's why I wrote this. If you have other suggestion to implement that let me know 👍

Reply:
:+1: OK